trl/import_utils.py

```diff
@@ -59,6 +61,18 @@ def is_vllm_available() -> bool:
     return _vllm_available
 
 
+def is_agents_available() -> bool:
+    return _langchain_experimental_available and _vllm_available and _e2b_available
+
```

Comment:
```suggestion
```

I would not combine requirements here. Let's be fine grained where the deps are used